src/data/items.txt

```diff
@@ -11616,6 +11616,20 @@
 11588	yam mayo	878600079	nutramekin.gif	food	t,d	11	ramekins of yam mayo
 11589	yam burrito	855430740	burrito.gif	food	t,d	11
 11590
-11591
-11592
+11591	mini kiwi egg	845486056	kiwi_egg.gif	grow	t	0
+11592	aviator goggles	628912777	goggles.gif	familiar	t,d	75
 11593	Thwaitgold Illinigina illinoiensis model	257510677	thwaitilli.gif	none		0
+11594	mini kiwi	940910612	kiwi_minikiwi.gif	food, paste, cook, mix	t,d	5
+11595	mini kiwi bikini	705305695	kiwi_bikini.gif	accessory	t,d	11
+11596	mini kiwitini	263511672	martini.gif	drink	t,d	11
+11597	mini kiwi invisible dirigible	718055433	kiwi_dirigible.gif	familiar	t,d	30
+11598	mini kiwi aioli	863064561	ramekin.gif	usable	t,d	11
+11599	mini kiwi silicon tiepin	701181594	kiwi_tiepin.gif	accessory	t,d	40
+11600	mini kiwi tipi	496300747	kiwi_tipi.gif	usable	t,d	11
+11601	mini kiwi digitized cookies	733008951	10101.gif	food	t,d	11
+11602	mini kiwi intoxicating spirits	774169937	beerbottle.gif	drink	t,d	11
+11603	mini kiwi antimilitaristic hippy petition	779505307	document.gif	potion, usable	t,d	11
```

Comment:
Did KoLmafia's item derivation say this is "potion, usable"?
That now means "this is a potion that you cannot multiuse".
The vast majority of potions are multiusable, so the default should be simply "potion".

I think you should make that the default - and also remove "usable" from the new potions - unless you have already determined that they are not multiusable.

Reply:
It was auto-generated, but it doesn't show up on multiuse.php, so that means in this case it indeed isn't multi-usable?

Reply:
```
> test newitem 459688276

Unknown item found: mini kiwi illicit antibiotic (11604, 459688276)
--------------------
11604 mini kiwi illicit antibiotic 459688276 kiwi_pill.gif potion, usable t,d 11
Item mini kiwi illicit antibiotic Effect: "Incredibly Healthy", Effect Duration: 5
--------------------
```

Yes. The "usable" is added by default.
Since "potion" implies "multiple" - UNLESS you manually add "usable" - the default should not include "usable".

Reply:
You mean that if you have multiple of them in inventory and visit inventory, KoL does not provide "use one" AND "use some"?

Which is more annoying - trying to multi use and having KoL fail it, or using multiple one at a time - and having them all succeed - but taking a lot more requests?

I guess this is good enough for now. :)